deep_speech_2/data_utils/augmentor/speed_perturb.py

```diff
@@ -0,0 +1,46 @@
+"""Contain the speech perturbation augmentation model."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from data_utils.augmentor.base import AugmentorBase
+
+
+class SpeedPerturbAugmentor(AugmentorBase):
+    """Augmentation model for adding speed perturbation.
+
+    See reference paper here:
+    http://www.danielpovey.com/files/2015_interspeech_augmentation.pdf
+
+    :param rng: Random generator object.
+    :type rng: random.Random
+    :param min_speed_rate: Lower bound of new speed rate to sample.
+    :type min_speed_rate: float
+    :param max_speed_rate: Upper bound of new speed rate to sample.
+    :type max_speed_rate: float
+    """
+
+    def __init__(self, rng, min_speed_rate, max_speed_rate):
+
+        if (min_speed_rate < 0.5):
```

Comment:
1. 0.5 or 0.9? It is inconsistent with the error message.
2. Please also add this explanation in function doc.